code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm

```diff
@@ -383,7 +383,7 @@
 	if(lava_cooldown > world.time)
 		return
 	var/turf/T = get_turf(user)
-	if(target && istype(target, /turf/open) && target in view(7,T))
+	if(target && istype(target, /turf/open) && (target in view(7,T)))
```

Comment:
shouldn't we just use (istype(target, /turf/open) && (target in view(7,T))?
Isn't the first target redundant due to the istype?